src/pages/Controllers/Controllers.test.js

```diff
@@ -58,4 +58,19 @@ describe("Controllers table", () => {
     );
     expect(wrapper.find("tbody tr").get(0)).toMatchSnapshot();
   });
+
+  it("renders a warning if you do not have access", () => {
```

Comment:
Can you add a test to check it does display the details if "superuser" or "admin"

Reply:
That's the default case, but I can add a comment in the other test cases to make that clear.